### ISSUE TITLE
Add attribute "mfau" in AlastriaToken

### ISSUE THREAD

**Issue by VictorNS69** [alastria/alastria-identity-lib#148]
MFAU (Multi Factor Authentication URL) is optional URL for MFA callbacks.

It improves security in stories like US 1.2 Identity Creation.

Wiki link: https://github.com/alastria/alastria-identity/wiki/Artifacts-and-User-Stories-Definitions#01-alastria-token-at

**Comment by VictorNS69** [comment]
Blocked by the Architecture team. 

As soon as it is well defined, we will unlock it.

**Comment by rober12** [comment]
New issue open more complete about this and more fields

https://github.com/alastria/alastria-identity-lib/issues/185
